G/gfm/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "gfm"
+version = v"29.0.6"
```

Comment:
I liked 0.29.6 better, it was somewhat closer to the upstream version

Reply:
But what do I do when up-upstream releases `0.29.1`, which is then built as `0.29.1-gfm.0` by upstream? Seems much less likely that anyone releases a 1.x :)

Reply:
People often do stuff like `100 * patch + else`, so that'd be `0.29.100`, and `0.29.2-gfm.5` would be `0.29.205`.

Reply:
For example https://github.com/JuliaPackaging/Yggdrasil/blob/23c4fc91c6c00e1dc91f3423e7ba80829cd4ce1d/S/Singular/build_tarballs.jl#L30-L34

Reply:
Ok, I'm convinced.